src/game/scenes/Game.ts

```diff
@@ -161,11 +176,11 @@ export class Game extends Phaser.Scene {
 			0,
 		);
 		if (this.controlKeys.right.isDown) {
-			this.player.flipX = false;
+			this.player.sprite.flipX = false;
 			movementDirection.x++;
 		}
 		if (this.controlKeys.left.isDown) {
-			this.player.flipX = true;
+			this.player.sprite.flipX = true;
 			movementDirection.x--;
 		}
 		if (this.controlKeys.up.isDown) {
```

Comment:
the logic pretaining to player movement/animation should be moved to the `PlayerCharacter` class's `update` method and called here with `this.player.update()`